src/index.js

```diff
@@ -1,6 +1,6 @@
 import mixer from './mixer';
-import isFunction from 'lodash/lang/isFunction';
 
+const isFunction = val => typeof val === 'function';
```

Comment:
Latest lodash already has check for generator functions, but I guess this is much easier :)